docs/README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  <a href="#">
```

Comment:
Why is this a link when it doesn't have an href?

Reply:
This is copy/paste from the primary README.md in our repository. 

I forget the exact reason why I did it this way (3 years ago: https://github.com/ionic-team/ionic-framework/commit/3fc278a5392181febe5bf35b35a58369f79995a5).

I think we can remove the link here and probably could do the same with the main README if we wanted.

Reply:
Not sure 🤔 I copied it from the main README here: https://github.com/ionic-team/ionic-framework/blob/main/README.md

Removed: https://github.com/ionic-team/ionic-framework/pull/29266/commits/80f8ff6e3d8a44632eaeb9b295a5aff9a5e15de1